lib/hanami/utils/hash.rb

```diff
@@ -48,7 +48,7 @@ def initialize(hash = {}, &blk)
         @hash.default_proc = blk
       end
 
-      # Convert in-place all the keys to Symbol instances, nested hashes are converted too.
+      # Convert in-place all the keys to Symbol instances. Nested hashes are converted too if deep argument is truthy.
```

Comment:
Like this: Convert in-place all the keys to Symbol instances, optionally nested hashes are converted too if deep argument is truthy. ?